### PR TITLE
chore(onyx-356): add AREnableSingleActivityPanelScreen feature flag

### PR DESCRIPTION
**Pull request by nickskalkin**
This PR resolves [ONYX-356]

### Description

[Jira ticket](https://artsyproduct.atlassian.net/browse/ONYX-356)
[Echo PR](https://github.com/artsy/echo/pull/539)

Add Single Activity Panel screen feature flag.

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- add AREnableSingleActivityPanelScreen feature flag - @nickskalkin 

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-356]: https://artsyproduct.atlassian.net/browse/ONYX-356?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by nickskalkin** [comment]
Moved to draft since related Echo PR is not merged

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (add AREnableSingleActivityPanelScreen feature flag - @nickskalkin)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0ab5c2c527e11a57a2c8a3ead6e19ae00d81b31a
</p>

**Review by olerichter00 — APPROVED** [review]

**Review by MounirDhahri — APPROVED** [review]
Nice

File: src/app/store/config/features.ts [hunk 1]
```diff
@@ -305,6 +305,12 @@ export const features: { [key: string]: FeatureDescriptor } = {
     showInDevMenu: true,
     echoFlagKey: "ARShowArtistsAlertsSet",
   },
+  AREnableSingleActivityPanelScreen: {
+    description: "Enable Single Activity Panel screen",
+    readyForRelease: false,
```
Comment (MounirDhahri) [inline src/app/store/config/features.ts:310]: I guess you don't want to release this yet. There is no need to do add the echo flag in that case btw

Comment (nickskalkin) [reply src/app/store/config/features.ts:310]: Yeah - that's what I though as well right after I've merged Echo. But in any case we probably will need it in future, so I'm going to merge

**Pull request merged**